test/unit_tests/modules/util/type-checkers-spec.js

```diff
@@ -0,0 +1,120 @@
+'use strict';
+var chai = require( 'chai' );
+var expect = chai.expect;
+var typeCheckers = require(
+  '../../../../src/static/js/modules/util/type-checkers.js'
+);
+
+var undefinedVar;
+var blankVar = '';
+var aString = 'bar';
+var aNum = 42;
+var aDate = new Date( 2011, 7, 21 );
+function aFunction() {
+  return true;
+}
+var anObject = {
+  a: '1',
+  b: '2',
+  c: '3'
+};
+var anArray = [ 1, 2, 3 ];
+
+describe( 'TypeCheckers isUndefined', function() {
+  it( 'should identify undefined variables', function() {
+    expect( typeCheckers.isUndefined( undefinedVar ) ).to.be.true;
```

Comment:
@anselmbradford @KimberlyMunoz Technically we can use [Mocha with Protractor](https://github.com/angular/protractor/blob/master/docs/frameworks.md), but it seems like a lot of work for a dubious benefit of assertion uniformity. 
TL;DR version mocha is a test framework for node, hence OAH is using it (not sure about  others), 
Jasmine is an older, more mature and more full-features framework (included spies, mocks and assertions) that is default for Protractor.